examples/quantization/brevitas/quantize_llm.py

```diff
@@ -29,7 +29,7 @@ def main(args):
         cpu_device_map=args.cpu_device_map,
     )
 
-    quantizer = BrevitasQuantizer.from_pretrained(args.model, device_map=args.device)
+    quantizer = BrevitasQuantizer.from_pretrained(args.model, device_map="cpu" if use_accelerate else args.device)
```

Comment:
If I'm correct, this does not change much to the example script since we have https://github.com/huggingface/optimum-amd/blob/939345932a5a95a638d1e00af50f527a79067ca4/examples/quantization/brevitas/quantize_llm.py#L62-L63, is that so? (just that the offloading is not done using accelerate native functions, but our patches ones)

Reply:
Yes, exactly. I thought this is where we landed in [this thread](https://github.com/huggingface/optimum-amd/pull/86#issuecomment-1964408165).

We also do not encounter these issues: [1](https://github.com/huggingface/optimum-amd/pull/86#discussion_r1501079645), [2](https://github.com/huggingface/optimum-amd/pull/86#issuecomment-1963636606)

I think this is simpler and doesn't break anything, but I'm open to feedback.